app_dart/lib/src/service/scheduler.dart

```diff
@@ -134,7 +134,11 @@ class Scheduler {
     final List<Tuple<Target, Task, int>> toBeScheduled = <Tuple<Target, Task, int>>[];
     for (Target target in initialTargets) {
       final Task task = tasks.singleWhere((Task task) => task.name == target.value.name);
-      final SchedulerPolicy policy = target.schedulerPolicy;
+      SchedulerPolicy policy = target.schedulerPolicy;
```

Comment:
I would've preferred to add this into target, but we don't pass the branch down to the target